rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -534,6 +534,8 @@ namespace rsx
 			return fmt::format("RSX [0x%07x]", +rsx->ctrl->get);
 		};
 
+		method_registers.init();
```

Comment:
this commit has nothing to do with this PR

Reply:
It unbreaks pure HLE mode.